src/PaymentPart/Output/HtmlOutput/HtmlOutput.php

```diff
@@ -0,0 +1,182 @@
+<?php
+
+namespace Sprain\SwissQrBill\PaymentPart\Output\HtmlOutput;
+
+use Sprain\SwissQrBill\PaymentPart\Output\AbstractOutput;
+use Sprain\SwissQrBill\PaymentPart\Output\Element\OutputElementInterface;
+use Sprain\SwissQrBill\PaymentPart\Output\Element\Placeholder;
+use Sprain\SwissQrBill\PaymentPart\Output\Element\Text;
+use Sprain\SwissQrBill\PaymentPart\Output\Element\Title;
+use Sprain\SwissQrBill\PaymentPart\Output\HtmlOutput\Template\PlaceholderElementTemplate;
+use Sprain\SwissQrBill\PaymentPart\Output\HtmlOutput\Template\PrintableStylesTemplate;
+use Sprain\SwissQrBill\PaymentPart\Output\HtmlOutput\Template\TextElementTemplate;
+use Sprain\SwissQrBill\PaymentPart\Output\HtmlOutput\Template\PaymentPartTemplate;
+use Sprain\SwissQrBill\PaymentPart\Output\HtmlOutput\Template\TitleElementTemplate;
+use Sprain\SwissQrBill\PaymentPart\Output\OutputInterface;
+use Sprain\SwissQrBill\PaymentPart\Translation\Translation;
+
+final class HtmlOutput extends AbstractOutput implements OutputInterface
```

Comment:
Wouldn't it be a better option to have a "HTML" service that builds such objects, instead of having all the logic in here? It has lots of responsibilities like building payment parts, replacing strings or translating content.
It would allow using specific services, with limited responsibilities, as dependencies.